RecoBTag/DeepBoostedJet/python/pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags_cfi.py

```diff
@@ -0,0 +1,136 @@
+import FWCore.ParameterSet.Config as cms
+
+pfMassDecorrelatedDeepBoostedDiscriminatorsJetTags = cms.EDProducer(
+   'BTagProbabilityToDiscriminator',
+   discriminators = cms.VPSet(
+      cms.PSet(
+         name = cms.string('TvsQCD'),
```

Comment:
assuming these are the "binarized" scores provided in the GitLab version, it would be nice to add the separate ZvsQCD, ZbbvsQCD, HbbvsQCD, H4qvsQCD discriminators that were provided there

Reply:
@kpedro88 These scores are added now: https://github.com/cms-sw/cmssw/pull/23768/files#diff-0b5067ac36f218c17fd59ded2a4272b6R39.